src/main/java/mate/academy/spring/model/dto/request/UserRequestDto.java

```diff
@@ -0,0 +1,28 @@
+package mate.academy.spring.model.dto.request;
+
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotNull;
+
+public class UserRequestDto {
+    @NotNull
+    @Email
+    private String email;
```

Comment:
NotNull here is redundant, also add Valid annotation where are you using this Dto